prod/sst/sst.config.ts

```diff
@@ -156,106 +155,146 @@ export default $config({
+                'curl -f http://localhost:4848/ || exit 1',
               ],
-            }),
-      },
-      transform: defu(EBS_TRANSFORM, BASE_TRANSFORM, {
-        target: {
-          stickiness: {
-            enabled: true,
-            type: 'lb_cookie',
-            cookieDuration: 120,
+              interval: '5 seconds',
+              retries: 3,
+              startPeriod: '300 seconds',
+            },
+            environment: {
+              ...commonEnv,
+              ZERO_CHANGE_STREAMER_URI: replicationManager.url,
+              ZERO_UPSTREAM_MAX_CONNS: '15',
+              ZERO_CVR_MAX_CONNS: '160',
+            },
+            logging: {
+              retention: '1 month',
+            },
           },
-          loadBalancingAlgorithmType: 'least_outstanding_requests',
-        },
-        autoScalingTarget: {
-          minCapacity: 1,
-          maxCapacity: 10,
+        ],
+
+        loadBalancer: {
+          public: true,
+          //only set domain if both are provided
+          ...(process.env.DOMAIN_NAME && process.env.DOMAIN_CERT
+            ? {
+                domain: {
+                  name: process.env.DOMAIN_NAME,
+                  dns: false,
+                  cert: process.env.DOMAIN_CERT,
+                },
+                ports: [
+                  {
+                    listen: '80/http',
+                    forward: '4848/http',
+                    container: 'view-syncer',
+                  },
+                  {
+                    listen: '443/https',
+                    forward: '4848/http',
+                    container: 'view-syncer',
+                  },
+                ],
+              }
+            : {
+                ports: [
+                  {
+                    listen: '80/http',
+                    forward: '4848/http',
+                  },
+                ],
+              }),
         },
-      }),
-      // Set this to `true` to make SST wait for the view-syncer to be deployed
-      // before proceeding (to permissions deployment, etc.). This makes the deployment
-      // take a lot longer and is only necessary if there is an AST format change.
-      wait: false,
-    });
+        transform: defu(EBS_TRANSFORM, BASE_TRANSFORM, {
+          target: {
+            stickiness: {
+              enabled: true,
+              type: 'lb_cookie',
+              cookieDuration: 120,
+            },
+            loadBalancingAlgorithmType: 'least_outstanding_requests',
+          },
+          autoScalingTarget: {
+            minCapacity: 1,
+            maxCapacity: 10,
+          },
+        }),
+        // Set this to `true` to make SST wait for the view-syncer to be deployed
+        // before proceeding (to permissions deployment, etc.). This makes the deployment
+        // take a lot longer and is only necessary if there is an AST format change.
+        wait: false,
+      },
+      {
+        apiKey: process.env.DATADOG_API_KEY,
```

Comment:
can we add service w/o otel if the datadog API key is missing? Just want to make sure users can deploy Zero without having to have datadog set up.